tasks/bottle.js

```diff
@@ -191,6 +191,7 @@ function bottle(versions, increment, forceNpm) {
 	const tagVersion = versions[1];
 	const bowerVersion = versions[2];
 	const packageVersion = versions[3];
+	const postBottleAdvice = `\nNote: To update this code in the app/s that use it, use 'nbt rebuild {app name}', or 'nbt rebuild --serves {service type}'`;
```

Comment:
There's a scope issue here, hence the failing build. Declare `let postBottleAdvice;` outside of the bottle function (i.e. in the main body of the module), then just set the val on line 194 without `const`. That should fix it.